src/ServicePulse.Host.Tests/Owin/StaticMiddlewareTests.cs

```diff
@@ -8,21 +8,21 @@ namespace ServicePulse.Host.Tests.Owin
     [TestFixture]
     public class StaticMiddlewareTests
     {
-        [Test]
-        public void Should_default_to_octetstream_mimetype()
-        {
-            var middleware = new StaticFileMiddleware(new DummyNext());
-            var context = new OwinContext
-            {
-                Request =
-                {
-                    Path = new PathString("/filename.unknown"),
-                    Method = "GET"
-                }
-            };
-            middleware.Invoke(context);
-            Assert.AreEqual(("application/octet-stream"), context.Response.ContentType);
-        }
+        //[Test]
```

Comment:
@mikeminutillo @janovesk is there any reason that this test was not removed?

Reply:
The intention was to put something back in it's place after the security advisory was published. Maybe an approval test of all extensions in the assembly along with the mime-type or something.